codacy/values-production.yaml

```diff
@@ -94,47 +94,47 @@ global:
     postgresqlPassword: <--- codacy-db-password --->
     host: <--- codacy-db-host --->
     service:
-      port: 5432
+      port: <--- codacy-db-port --->
```

Comment:
Do you need to do this? Isn't the port the default one? Can't we just "not replace" this one?